.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,3 @@
+# Require that new issues use a template.
+# See: https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
```

Comment:
is there a way to turn this off for any repos? I'm not sure we can set this for all of `cyberark` github

Reply:
I'm not sure if the contents of this `ISSUE_TEMPLATE` is an all or nothing proposition.  I can temporarily drop this change from the PR and play with it in the playroom repo to see how it works before we add it here.  WDYT?